src/deploy/functions/prompts.ts

```diff
@@ -25,21 +27,46 @@ export async function promptForFailurePolicies(
   if (failurePolicyFunctions.length === 0) {
     return;
   }
-  const failurePolicyFunctionLabels = failurePolicyFunctions.map((fn: CloudFunctionTrigger) => {
-    return getFunctionLabel(fn.name);
+
+  // N.B. Because context is an any, we need to store it in a temporary. This
+  // lets us do a 'for of' loop later with typeinfo.
+  const existingFunctions = await gcp.cloudfunctions.listAllFunctions(context.projectId);
```

Comment:
We already make this call once during the deployment process, at the start of 'deploy.ts' - WDYT about reordering things such that we only need to do this once?